region/maridia/inner-yellow.json

```diff
@@ -416,6 +416,48 @@
                     "This is applicable using the Power Bomb block platform in PCJR or the normal platform in Tourian Escape Room 2."
                   ]
                 },
+                {
+                  "name": "Cross Room Jump - Spring Ball Jump Above",
+                  "notable": false,
+                  "requires": [
+                    "canCrossRoomJumpIntoWater",
+                    {"adjacentJumpway": {
+                      "jumpwayType": "platformBelow",
+                      "fromNode": 2,
+                      "minHeight": 7,
+                      "maxHeight": 7,
+                      "maxLeftPosition": 1,
+                      "minRightPosition": 1
+                    }},
+                    "canCrouchJump",
+                    "canSpringBallJumpMidAir"
+                  ],
+                  "note": [
+                    "Standing on the platform in the room below, crouch jump up and press right against the door frame through the transition.",
+                    "After the transition, perform a spring ball jump mid-air to reach the ledge."
+                  ]
+                },
+                {
+                  "name": "Cross Room Jump - Spring Ball Jump Below",
+                  "notable": false,
+                  "requires": [
+                    "canCrossRoomJumpIntoWater",
+                    {"adjacentJumpway": {
+                      "jumpwayType": "platformBelow",
+                      "fromNode": 2,
+                      "minHeight": 8,
+                      "maxHeight": 10,
+                      "maxLeftPosition": 1,
+                      "minRightPosition": 1
+                    }},
+                    "canSpringBallJumpMidAir"
```

Comment:
Feels like a `can3HighMidAirMorph`.  
I was using the pause to time the springball jump.

Reply:
IBJ into springball jump could work instead of a quick morph, but I don't really like it.

Reply:
Added a `can3HighMidAirMorph` requirement now for the height 8 case.